work11/index.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const div = document.querySelector(".wrap")
```

Comment:
nits
I found it better to use getElementById.
This has been discussed in discode before, and at that time, Monaka-san, a student of mine, gave me a reference link. -> [this](https://whatabouthtml.com/difference-between-getelementbyid-and-queryselector-180)

I believe that using getElementById is faster and less buggy from a performance standpoint.

Reply:
Thank you for the article.

I understand that getElementById is fast, but in some cases you should also use querySelector.

This time, I will touch the ID, so I corresponded there.

please check!

commit number
bd0d099